qipr_approver/approver/urls.py

```diff
@@ -4,7 +4,7 @@
 
 urlpatterns = [
     url(r'^$', views.index, name='index'),
-    url(r'^shib/$', views.fake_shib, name='shib'),
+    url(r'^shib/$', views.shib_login, name='shib'),
```

Comment:
I almost think we ought to have a real shib and fake shib route. We still have all the fake shib stuff in here for local development and it would be good to keep in clear that they are separate? I don't know how needed that is though and it would take a minute to implement. Lets think about it later if someone is confused